docs/reference/environment.md

```diff
@@ -531,10 +531,11 @@ Equivalent to the
 
 Managed Python installations information is hardcoded in the `uv` binary.
 
-This variable can be set to a URL pointing to JSON to use as a list for Python installations.
-This will allow for setting each property of the Python installation, mostly the url part for offline mirror.
+This variable can be set to a local path or file://, http://, or / https:// URL pointing to
```

Comment:
```suggestion
This variable can be set to a local path or URL pointing to
```

(I don't think the specificity is needed -- also there's an extra `/` here if you want to preserve this text)